src/lang/std/sketchConstraints.ts

```diff
@@ -43,6 +43,16 @@ export function getSketchSegmentFromSourceRange(
       index: number
     }
   | Error {
+  const lineIndex = sketchGroup.value.findIndex(
+    ({ __geoMeta: { sourceRange } }: Path) =>
+      sourceRange[0] <= rangeStart && sourceRange[1] >= rangeEnd
+  )
+  const line = sketchGroup.value[lineIndex]
+  if (line)
+    return {
+      segment: line,
+      index: lineIndex,
+    }
```

Comment:
Could you put braces around this `if`? I know we've had times where missing an early return like this caused some confusion while debugging.

Reply:
I'll change this one, but return without braces is used a tone in the app.

How'd you miss it, like it's harder to parse or something?

Reply:
Yeah Frank is right, this has caused confusion for one reason or another. I don't know exact examples on hand but I swear it's happened too. I think when `return` is on the same line as the `if`, it's just easier to understand, but when it's on separate lines? Always seem to mess one of us up

Reply:
Yeah okay.

I wonder if there's a fmt rule we can turn on to change them all, it would be nice if it had that nuance, add braces if return is on separate line, if not, it's fine.

Reply:
probably not worth cycles figuring out how to do the above though.